### DISCUSSION TITLE
Google search console: sitemap-index discovered urls 0

### DISCUSSION THREAD

**Discussion by kdichev** [gatsbyjs/gatsby#35958] [Help]
Hi guys I am using `gatsby-plugin-sitemap` to generate my `sitemap.xml`. I have only two pages but the pluging creates `sitemap-index.xml` that refers to `sitemap-0.xml`. That's okay I guess bit looks like Google Search Console is not picking up the `sitemap-0.xml` and it's not discovering my pages. When I do a URL inspection, my index page is in state `
Indexed, not submitted in sitemap`. This is how I use the plugin:

```
{
  resolve: `gatsby-plugin-sitemap`,
  options: { excludes: ["/thank-you"] },
},
 ```

Do you have an idea how I can fix that, I kind of think this might hurt my SEO but I am not sure, but would love to fix this.

![image](https://user-images.githubusercontent.com/12297753/174624604-30b37c23-bb96-4133-89b5-3309faacf055.png)
![image](https://user-images.githubusercontent.com/12297753/174625075-9ef2db9d-b349-41bf-bf55-3f78c4d1e7ca.png)
![image](https://user-images.githubusercontent.com/12297753/174625010-16ac9d55-4cb0-4de5-8809-cbe94a04265d.png)

**Comment by LekoArts** [comment]
Maybe you don't allow Google bots to crawl your site? (e.g. in `robots.txt`). While I have configured my plugin a little different (set the `output` option to `/`) it crawls my sitemap:

![image](https://user-images.githubusercontent.com/16143594/174723755-6477f2d5-a8af-48aa-b282-5948a6a8addd.png)

It then should also show the `sitemap-0.xml` in the overview.

**Reply by kdichev** [reply]
I have `gatsby-plugin-robots-txt` in my config which outputs to:

```
User-agent: *
Allow: /
Sitemap: https://www.vililacosta.com/sitemap/sitemap-index.xml
Host: https://www.vililacosta.com
```
